cdap-ui/app/features/adapters/etlapps.less

```diff
@@ -125,25 +122,317 @@ body.theme-cdap.state-adapters-create {
 
   .modal {
 
+    .modal-dialog {
+      margin-top: 115px;
+    }
     .modal-lg {
       width: 95%;
     }
+    .modal-header {
+      background-color: @body-bg;
+      color: @cdap-header;
+      height: 60px;
+      padding: 0;
+      .border-radius(4px 4px 0 0);
+      div.pull-left {
+        height: 60px;
+        padding-left: 15px;
+        max-width: 725px;
```

Comment:
I believe this would cause issues if the browser is resized. It be might be better if we use the bootstrap grid here. On a laptop full browser size it looks perfect but should scale atleast for md screen size. Thoughts?